.github/FUNDING.yml

```diff
@@ -0,0 +1,2 @@
+github: VirtualMaker
+patreon: RageAgainstThePixel
```

Comment:
```suggestion
```

Reply:
We could probably get rid of this file altogether

Reply:
Removed 🫡